File: app/src/main/java/com/kota/ASFramework/PageController/ASListViewController.java
package com.kota.ASFramework.PageController;

import android.database.DataSetObservable;
import android.database.DataSetObserver;
import android.widget.ListAdapter;

/* loaded from: classes.dex */
public abstract class ASListViewController implements ListAdapter {
  private final DataSetObservable mDataSetObservable = new DataSetObservable();

  @Override // android.widget.Adapter
  public void registerDataSetObserver(DataSetObserver observer) {
    this.mDataSetObservable.registerObserver(observer);
  }

  @Override // android.widget.Adapter
  public void unregisterDataSetObserver(DataSetObserver observer) {
    this.mDataSetObservable.unregisterObserver(observer);
  }

  public void notifyDataSetChanged() {
    this.mDataSetObservable.notifyChanged();
  }
}

File: app/src/main/java/com/kota/asFramework/pageController/ASListViewController.kt
package com.kota.asFramework.pageController

import android.database.DataSetObservable
import android.database.DataSetObserver
import android.widget.ListAdapter

/* loaded from: classes.dex */
abstract class ASListViewController : ListAdapter {
    private val mDataSetObservable = DataSetObservable()

    // android.widget.Adapter
    override fun registerDataSetObserver(observer: DataSetObserver?) {
        this.mDataSetObservable.registerObserver(observer)
    }

    // android.widget.Adapter
    override fun unregisterDataSetObserver(observer: DataSetObserver?) {
        this.mDataSetObservable.unregisterObserver(observer)
    }

    fun notifyDataSetChanged() {
        this.mDataSetObservable.notifyChanged()
    }
}
